backend/src/main/java/heartbeat/controller/report/GenerateReportController.java

```diff
@@ -68,12 +52,75 @@ public InputStreamResource exportCSV(@PathVariable String dataType, @PathVariabl
 	@GetMapping("/{reportId}")
 	public ResponseEntity<ReportResponse> generateReport(@PathVariable String reportId) {
 		boolean generateReportIsOver = generateReporterService.checkGenerateReportIsDone(reportId);
+		ReportResponse reportResponse = generateReporterService.getComposedReportResponse(reportId,
+				generateReportIsOver);
 		if (generateReportIsOver) {
-			ReportResponse reportResponse = generateReporterService.getReportFromHandler(reportId);
 			log.info("Successfully generate Report, report id: {}, reports: {}", reportId, reportResponse);
+			generateReporterService.generateCSVForMetric(reportResponse, reportId);
 			return ResponseEntity.status(HttpStatus.CREATED).body(reportResponse);
 		}
-		return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
+		return ResponseEntity.status(HttpStatus.OK).body(reportResponse);
+	}
+
+	@PostMapping("/board")
+	public ResponseEntity<CallbackResponse> generateBoardReport(@RequestBody GenerateBoardReportRequest request) {
+		log.info(
+				"Start to generate Board Report, metrics: {}, consider holiday: {}, start time: {}, end time: {}, board-report id: {}",
```

Comment:
1.Board report to lower case
2.need to show request in log

Reply:
lower case is done